cava/samples/onnxruntime/onnx_dump.c

```diff
@@ -5,6 +5,7 @@ ava_number(10);
 ava_cflags(-I/usr/local/cuda-10.1/include -I../headers);
 ava_libs(-L/usr/local/cuda-10.1/lib64 -lcudart -lcuda -lcublas -lcudnn -lcufft -lcurand -lcusparse -lcusolver);
 ava_export_qualifier();
+ava_soname(cudart);
```

Comment:
I'm not leaning to specify soname for onnxrt and tf guestlib. As shown in line 6, they are collections of annotations for `-lcudart -lcuda -lcublas -lcudnn -lcufft -lcurand -lcusparse -lcusolver`.

Reply:
PS: Changing soname would require updating the spec's CMakeLists.txt as well.

Reply:
ah, it already generates libcudart.so anyways so this isn't necessary like it was for cudadrv